apps/demos/Demos/DataGrid/SimpleArray/Angular/app/app.component.ts

```diff
@@ -16,8 +16,12 @@ if (!/localhost/.test(document.location.host)) {
 export class AppComponent {
   customers: Customer[];
 
+	columns: string[];
+
```

Comment:
Let's make the indent like in our code style (smaller)
Also, let's initialize the ```columns``` class field in an inline way:
```suggestion
  columns = ['CompanyName', 'City', 'State', 'Phone', 'Fax'];
```